paywall/src/static/adremover/prod.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
+    <meta charset="utf-8" />
+    <title>Lorem Ipsum Daily</title>
+    <script>
+      ;(function(d, s) {
+        var js = d.createElement(s),
+          sc = d.getElementsByTagName(s)[0]
+        js.src = '/static/unlock.1.0.min.js'
+        sc.parentNode.insertBefore(js, sc)
+      })(document, 'script')
+    </script>
+
+    <script>
+      var unlockProtocolConfig = {
+        locks: {
+          '0x2BDb87333b15E20d948BC5235569f0cA3E6c707C': {
+            name: 'One Day',
+          },
+          '0x1b75BC108a0259355475CE45CdFB5599361abcbe': {
+            name: 'A Good Deal!',
+          },
+        },
+        icon: 'http://lofrev.net/wp-content/photos/2017/05/logo_black.png',
```

Comment:
Should we avoid hotlinking to a third-party image host? Not a huge deal, but in the event that site goes down we don't want to mistakenly think there's a bug in the paywall

Reply:
I think this is right! Let;'s use a data uri here @cellog ?